sbncode/CAFMaker/cafmakerjob_sbnd.fcl

```diff
@@ -31,7 +31,7 @@ services:
   RandomNumberGenerator: {} #ART native random number generator
   TFileService: { fileName: "fullchain_production_hist_prod5.root" closeFileFast: false }
 #  scheduler:    { wantTracer: false wantSummary: true }
-  message:      { debugModules: ["*"] destinations: { debugmsg:{type: "cout" threshold: "INFO"} } }
+  message:      { debugModules: ["*"] destinations: { debugmsg:{type: "cout" threshold: "INFO"  categories:{BackTracker: { limit: 0 }} } } }
```

Comment:
Since we are here: `debugModules: ["*"]` has been neutered ages ago (see e.g. `lar --print-description message`). It can be removed.

Reply:
Not using `debugModules` anymore.